yarn-project/bb-prover/src/avm_proving.test.ts

```diff
@@ -282,7 +282,14 @@ const proveAndVerifyAvmTestContract = async (
   // Then we test VK extraction.
   const succeededRes = proofRes as BBSuccess;
   const verificationKey = await extractVkData(succeededRes.vkPath!);
-  expect(verificationKey.keyAsBytes).toHaveLength(16);
+
+  // VK is composed of
+  // - circuit size encoded as a fr field element (32 bytes)
+  // - num of inputs encoded as a fr field element (32 bytes)
+  // - 16 affine elements (curve base field fq) encoded as fr elements takes (16 * 4 * 32 bytes)
+  // 16 above refers to the constant AvmFlavor::NUM_PRECOMPUTED_ENTITIES
+  // Total number of bytes = 2112
+  expect(verificationKey.keyAsBytes).toHaveLength(2112);
```

Comment:
This will be a pain, but I guess it's ok because it only changes if we add _precomputed_ columns. However, if we keep this assertion, maybe better to express it as

```
const NUM_PRECOMPUTED_ENTITIES = xxx;
expect(..... NUM_PRECOMPUTED_ENTITIES ...);
```

so that we can more easily update things.